### ISSUE TITLE
pip-review wants to upgrade modules to older versions?

### ISSUE THREAD

**Issue by teksec** [nvie/pip-tools#123]
Hi, i notice if i run: 'pip-review --interactive' that there are issues comparing the version numbers of modules already installed on my system.

Updates are shown that are older than what i already have installed. 

I believe pip-review should take in to account that it is possible to have a version number greater than what is available on the pypi repository (and if so, do not recommend an Upgrade)  =)

Examples:

PIL==1.1.6 is available (you have 1.1.7)
Upgrade now? [Y]es, [N]o, [A]ll, [Q]uit

ctypeslib==0.5.4a is available (you have 0.5.6)
Upgrade now? [Y]es, [N]o, [A]ll, [Q]uit

fuse-python==0.2 is available (you have 0.2.1)
Upgrade now? [Y]es, [N]o, [A]ll, [Q]uit

lightblue==0.2 is available (you have 0.3.2)
Upgrade now? [Y]es, [N]o, [A]ll, [Q]uit

louis==1.3 is available (you have 2.4.1)
Upgrade now? [Y]es, [N]o, [A]ll, [Q]uit

pcapy==0.10.3 is available (you have 0.10.8)
Upgrade now? [Y]es, [N]o, [A]ll, [Q]uit

psycopg2==2.5.4 is available (you have 2.4.5)
Upgrade now? [Y]es, [N]o, [A]ll, [Q]uit

python-apt==0.7.8 is available (you have 0.8.8.2)
Upgrade now? [Y]es, [N]o, [A]ll, [Q]uit

python-debianbts==1.10 is available (you have 1.11)
Upgrade now? [Y]es, [N]o, [A]ll, [Q]uit

volatility==2.1 is available (you have 2.4)
Upgrade now? [Y]es, [N]o, [A]ll, [Q]uit

wxPython-common==2.6.3.3 is available (you have 2.8.12.1)
Upgrade now? [Y]es, [N]o, [A]ll, [Q]uit

--------------------------------------
My system is Debian.
pip 1.5.6 from /usr/local/lib/python2.7/dist-packages (python 2.7)
pip-tools Version: 0.3.5

**Comment by digital-carver** [comment]
Just adding another instance of this issue: 

```
...
louis==1.3 is available (you have 2.3.0)
...
oneconf==0.0.1.dev0 is available (you have 0.2.8.1)
PAM==0.1.4 is available (you have 0.4.2)
PIL==1.1.6 is available (you have 1.1.7)
...
```

This keeps me from being able to use the `--auto` flag at all, and having to go through everything interactively to make sure downgrades don't happen.

**Comment by nvie** [comment]
pip-review only exists in < 1.0 versions of pip-tools, and will not be continued development.
